src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/CreateOrgInput.tsx

```diff
@@ -0,0 +1,63 @@
+import React, {FC, useState} from 'react'
+import {
+  ComponentSize,
+  ComponentStatus,
+  Form,
+  Input,
+} from '@influxdata/clockface'
+
+// Types
+import {OrgOverlayValidationError} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/CreateOrganizationOverlay'
+
+// Style
+import './CreateOrgInput.scss'
+
+const formLabel = 'Organization Name'
+
+interface Props {
+  handleValidateOrgName: (orgName: string) => void
+  orgName: string
+  validationMsg: OrgOverlayValidationError
+}
+
+export const CreateOrgInput: FC<Props> = ({
+  handleValidateOrgName,
+  orgName,
+  validationMsg,
+}) => {
+  const [userTouchedFormInput, setUserTouchedFormInput] = useState(false)
+
+  const handleInputChange = (evt: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
I hear you, and good idea, but I would use `evt` over `event` because `event` is a specialized term in the codebase for the eventing function that sends events to snowflake. If we rename regular handler events to `event`, that'll make those hard to distinguish when someone is running a Control+F searching just for the `event` calls that handle reporting.